bundles/GraphicServiceOrder/Resources/translations/validators.da.yaml

```diff
@@ -4,4 +4,8 @@ service_order_form:
       constraint:
         not_null: Angiv debitornummer eller anvend markedsføringskontoen hvis du kommer fra Borgerservice og biblioteker
         min: Debitornummer skal være præcis 4 cifre
-        max: Debitornummer skal være præcis 4 cifre
\ No newline at end of file
+        max: Debitornummer skal være præcis 4 cifre
+  delivery:
+    postal_code:
+      constraint:
+        not_null: Angiv et validt postnummer
```

Comment:
Missing newline at end of file.

Reply:
“gyldigt” may be a better word than “validt”.

Reply:
changed